snippets/quick-order-list.liquid

```diff
@@ -1,13 +1,28 @@
+{% comment %}
+  Renders a list of product's variants
+
+  Accepts:
+  - product: {Object} Product Liquid object
+  - show_image: {Boolean} Shows image of the variant in the row
+  - is_modal: {Boolean} Defines if this snippet lives in a modal (optional)
+  - id: {String} Adds a class to the snippet
```

Comment:
This doesn't actually add a class to the snippet but I'm not sure it's actually needed in the first place anyway. I shared why I think so in another comment.